internal/bot/bot.go

```diff
@@ -214,6 +219,23 @@ func (b *Bot) incrementUserMessageCount(message *structs.Message) {
 	}
 }
 
+func (b *Bot) sendCheckResultMessage(action string, message *structs.Message, processed structs.SpamCheckResult, deletedTime time.Time, channelId int64) {
```

Comment:
var-naming: method parameter channelId should be channelID (revive)

linter is a little bit angry :)